mayday.go

```diff
@@ -56,21 +57,29 @@ func openFile(f File) (*mayday.MaydayFile, error) {
 }
 
 func main() {
+	pflag.StringP("config-dir", "c", "/etc/mayday", "directory containing configuration file")
 	pflag.BoolP("danger", "d", false, "collect potentially sensitive information (ex, container logs)")
+	pflag.Parse()
 
 	viper.SetConfigName("config")
-	viper.AddConfigPath("/etc/mayday")
-	viper.AddConfigPath(".")
+	viper.AddConfigPath(pflag.Lookup("danger").Value.String())
 
 	err := viper.ReadInConfig()
 	if err != nil {
-		log.Fatal("Fatal error reading config: %s \n", err)
+		// viper returns an `unsupported config type ""` error if it can't find a file
+		// https://github.com/spf13/viper/issues/210
+		if strings.HasSuffix(err.Error(), `Type ""`) {
+			log.Printf("Could not find configuration file in /etc/mayday/config.json " +
+				"or in working directory.")
+			os.Exit(1)
+		}
+		log.Printf("Error reading configuration file.")
+		log.Printf("Fatal error reading config: %s \n", err)
```

Comment:
change to `log.Fatal("Fatal error reading config: %s\n", err)`?

extra print is redundant, log.Fatal calls os.Exit for you, and there's an extra space here.